ui-packages/packages/common/src/components/Molecules/DomainExplorerTable/DomainExplorerTable.tsx

```diff
@@ -58,7 +60,11 @@ interface IOwnProps {
   parameters: object[];
   rows: RowContent[];
   selected: string[];
+  setOrderByObj: (orderObj: object) => void;
   setRows: (rows: ((row: object[]) => object[]) | object[]) => void;
+  setRunQuery: (runQuery: boolean) => void;
+  setSortBy: (sortBy) => void;
```

Comment:
probably the same type as below, but to be sure.
```suggestion
  setSortBy: (sortBy: object) => void;
```

Reply:
@jstastny-cz done